docs/topics/proc_using-openrewrite-recipes.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+
+link:https://docs.openrewrite.org/[OpenRewrite], which is integrated into the {ProductShortName} CLI, uses recipes to automate large-scale, distributed source code refactoring.
+
+You can use OpenRewrite in the {ProductShortName} CLI to prepare Java applications for migration.
+
+[IMPORTANT]
+====
+OpenRewrite recipe support is provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
+====
+
+The `JavaxtoJakarta` recipe renames imported `javax` packages to their `jakarta` equivalents.
+
+.Procedure
+
+. Specify the OpenRewrite recipe when you run `mta-cli`:
++
+[source,terminal]
+----
+$ ./mta-cli --openrewrite "-DactiveRecipes=<recipe name>"  \
+  --input /path/to/source/project --goal dryRun
+----
+
+* `--openrewrite`: Flag specifying to run an OpenRewrite migration, instead of an MTA analysis.
+
+* `"-DactiveRecipes=<recipe name>"` : The OpenRewrite recipe to apply to the input project. `org.jboss.windup.JavaxtoJakarta` is the default shipped recipe but you can add your own to the shipped `rewrite.yml` and run that recipe instead. The `rewrite.yml file` is located in the `rules/openrewrite/`` folder in the uncompressed MTA distribution.
+
+* `--input`: The application to be evaluated.
+
+* `--goal`: Optional: The OpenRewrite Maven goal to run. Parameters:
+** `dryRun` : The script returns a list of proposed changes. By default, the output contains a message – `"Run 'mvn rewrite:run' to apply the recipes"` – that you should ignore.
+** `run`: The script makes all changes automatically without listing them first.
+
+. To apply the recipe, run the command using the `run` parameter for the `goal` argument:
++
+[source,terminal]
+----
+$ ./mta-cli --openrewrite "-DactiveRecipes=<recipe name>"  \
+  --input /path/to/source/project --goal run
```

Comment:
user-defined values should be in pointed brackets. </path/to/source/project> 
Is this an absolute or a relative path? It should be stated somewhere in the CLI guide.

`run` parameter for the `goal` argument - Looks like it should be `run` value for the `goal` parameter or option.

Reply:
But if the doc uses parameter and argument this loosely in other parts of the doc, no need to change.

Reply:
"/path/to/source/project" is from the SME. I can't find this or a similar path name including the word "source" in the repo, but if there's a better way to write it, please let me know. I added the <>s and made the other changes.